wolfcrypt/src/blake2s.c

```diff
@@ -424,6 +424,24 @@ int wc_InitBlake2s(Blake2s* b2s, word32 digestSz)
 }
 
 
+/* Init Blake2s digest with key, track size in case final doesn't want to "remember" */
+int wc_InitBlake2s_WithKey(Blake2s* b2s, word32 digestSz, const byte *key, word32 keylen)
+{
+    if (b2s == NULL){
+        return -1;
```

Comment:
BAD_FUNC_ARG is probably better here.

Reply:
No doubt.  blake2s.c and blake2b.c were both set up that way, and indeed the #include for error-crypt.h was missing.  I took a few minutes to fix them both up.